assets/queries/k8s/k8s_dashboard_integration/query.rego

```diff
@@ -0,0 +1,34 @@
+package Cx
+
+CxPolicy [ result ] {
+   document := input.document
+   metadata := document[i].metadata
+   kind := document[i].kind
+   check_kind(kind)
```

Comment:
I think you shouldn't restrict to "Deployment" because there could be a "Pod" that also has a container that deploys k8s dashboard. You can remove this line